static/src/stylesheets/module/_adslot.scss

```diff
@@ -528,3 +528,41 @@
     }
 
 }
+
+.mobilesticky-container {
+    position: fixed;
+    bottom: 0;
+    width: 320px;
+    margin: 0 auto;
+    right: 0;
+    left: 0;
+    z-index: $zindex-ads;
+    @include mq(phablet) {
+        display: none;
+    }
+}
+
+.ad-slot__closer {
+    display: none;
+    position: absolute;
+    right: 3px;
+    top: 1px;
+    height: $mpu-ad-label-height - 3px;
+    width: $mpu-ad-label-height - 3px;
+}
+
+.ad-slot--mobile-sticky .ad-slot__label .ad-slot__closer {
+    display: block;
+}
+
+.ad-slot__closer__x {
+    stroke: #121212;
```

Comment:
Can we use a colour code from the `_vars.scss` file? The colours in here should cover the entire palette used on the site.

Reply:
👍